code/modules/uplink/uplink_items.dm

```diff
@@ -1278,13 +1279,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/card/emag
 	cost = 6
 
-/datum/uplink_item/device_tools/emag
-	name = "Blue Space Cryptographic Sequencer"
-	desc = "The cryptographic sequencer, electromagnetic card, or emag, is a small card that unlocks hidden functions,Comes with a remote use function \
-			in electronic devices, subverts intended functions, and easily breaks security mechanisms."
-	item = /obj/item/card/emag/bluespace
-	cost = 7
-
```

Comment:
This change is out of scope

Reply:
left over from another pr removed it.

Reply:
But you didn't

Reply:
I don't know what i did wrong but in the uplink file its not there. Maybe i am missing what you are talking about.